docs/content/operations/recommendations.md

```diff
@@ -5,6 +5,39 @@ layout: doc_page
 Recommendations
 ===============
 
+# Some Genearl guidelines
+
+JVM Flags..
```

Comment:
Two dots is kind of strange. Suggest a colon or making this a subheading.

Reply:
using  a colon